__tests__/app.test.js

```diff
@@ -299,3 +299,54 @@ describe("4. GET /api/users", () => {
   });
   });
 
+  describe('POST /api/articles/:article_id/comments', () => {
+    test("status:201, responds with comment newly added to the comments database", () => {
+  
+      const newComment = {
+        username : "lurker",
+        body: 'lurking around'
+      };
+      return request(app)
+        .post(`/api/articles/2/comments`)
+        .send(newComment)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.comment).toEqual({
+            article_id: expect.any(Number),
+            author: 'lurker',
+            body: 'lurking around',
+            comment_id: expect.any(Number),
+            votes: 0,
+            created_at: expect.any(String) 
+          });
+        });
+    });
```

Comment:
Needs error handling for a lot of things here. We should always test errors for endpoints where the user has control over something;
- the user could pass an article id that doesnt exist
- the user could pass an article id that isnt a number
- the comment object could have missing/malformed properties
- the comment object could have a username that doesnt exist in the database